src/rest/AnalyticsEndpoint.ts

```diff
@@ -93,22 +95,40 @@ export class AnalyticsEndpoint {
     return this.getFromService<string[]>(url, params);
   }
 
+  public forgetVisitorId() {
+    Cookie.erase('visitorId');
+  }
+
+  public kill() {
```

Comment:
I think `disable() / disabled` instead of `kill() / dead` terminology would be more in line with the rest of the codebase to explain properly what it does.

Also, it's a bit less violent :D

Reply:
This is resolved as I removed the ability to disable [AnalyticsEndpoint](../tree/master/src/rest/AnalyticsEndpoint.ts) directly.